parts/seatbelt.go

```diff
@@ -0,0 +1,30 @@
+package parts
+
+import "fmt"
+
+type SeatbeltState string
+
+var (
+	SeatbeltStateEngauged    SeatbeltState = "engauged"
+	SeatbeltStateDisengauged SeatbeltState = "disengauged"
+)
+
+type Seatbelt struct {
+	State *SeatbeltState
```

Comment:
Why a pointer here?

Reply:
It is used to default the seatbelt to desengauged in the getState  function when it is nil